analyze_projects_multi_agent_v2.py

```
@@ -87,15 +87,59 @@
+            
+            self.model_usage[model]['calls'] += 1
+            self.model_usage[model]['cost'] += cost
+    
+    def print_session_summary(self, project_name=None):
+        """Print session cost summary"""
+        session_time = time.time() - self.session_start
+        
+        if project_name:
+            print(f"\n💰 Cost Summary for {project_name}:")
+        else:
+            print(f"\n💰 Session Cost Summary:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)